app/src/main/res/layout/hints_summary.xml

```diff
@@ -95,21 +94,20 @@
       android:id="@+id/hint_list_container"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:layout_marginStart="40dp"
-      android:layout_marginEnd="40dp"
-      android:layout_marginTop="8dp"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="68dp"
+      android:layout_marginEnd="28dp"
       android:layout_marginBottom="32dp"
+      app:layout_constraintTop_toTopOf="parent"
       android:orientation="vertical"
       android:visibility="@{isListExpanded? View.VISIBLE : View.GONE}">
 
       <TextView
         android:id="@+id/hints_and_solution_summary"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
         android:layout_gravity="start"
-        android:fontFamily="sans-serif"
-        android:textColor="@color/oppiaPrimaryText"
-        android:textSize="16sp" />
+        style="@style/Body"
+        android:textColor="@color/oppiaPrimaryText" />
     </LinearLayout>
-  </LinearLayout>
+  </androidx.constraintlayout.widget.ConstraintLayout>
 </layout>
+
```

Comment:
Remove this extra blank line.

Reply:
I removed the extra line. Now the file ends at line 112

Reply:
Actually that extra blank line is still there.